docs/eventing/channel-based-broker.md

```diff
@@ -134,21 +134,23 @@ installed with Knative Eventing
 The `ServiceAccount` was created two commands prior. The `RoleBinding` is
 created with this command.
 
-Create RBAC permissions granting access to shared configmaps for logging,
-tracing, and metrics configuration.
-
-_These commands assume the shared Knative Eventing components are installed in
```

Comment:
Keep this sentence, unless the guidance about system namespace is also mentioned elsewhere in these instructions (I can only see about 20 lines)

Reply:
I moved this to the beginning of this doc.